firefox-webext/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
+  "manifest_version": 2,
+  "name": "webcompat.com reporter",
+  "description": "Report website compatibility issues at webcompat.com",
+  "version": "0.2",
+  "applications": {
+    "gecko": {
+      "id": "miket@mozilla.com",
+      "strict_min_version": "47.0.0"
+    }
+  },
+  "background": {
+    "scripts": ["background.js"]
+  },
+  "permissions": [
+    "tabs",
+    "activeTab",
+    "https://webcompat.com/",
+    "<all_urls>"
+  ],
+  "icons": {
+    "32": "icon32.png",
+    "48": "icon48.png",
+    "64": "icon64.png",
+    "128": "icon128.png"
+  },
+  "browser_action": {
+    "default_icon": {
+      "19": "icon32.png",
```

Comment:
Are all these icons necessary? I added those that were both available in the chrome and firefox addons. However, in testing Nightly seems to require "19" on desktop or will not display an icon.